module/Motion/Rotation.h

```diff
@@ -0,0 +1,55 @@
+/**
+ * @file   Rotation.h
+ * @brief  回頭動作の中間クラス
+ * @author desty505
+ */
+
+#ifndef ROTATION_H
+#define ROTATION_H
+
+#include "Motion.h"
+#include "Mileage.h"
+#include "SpeedCalculator.h"
+#include "SystemInfo.h"
+
+class Rotation : public Motion {
+ public:
+  /**
+   * コンストラクタ
+   * @param _targetValue 回転角度(deg) 0~360
+   * @param _targetSpeed 目標速度
+   * @param _isClockwise 回頭方向 ture:時計回り, false:反時計回り
+   */
+  Rotation(int _targetValue, double _targetSpeed, bool _isClockwise);
+
+  /**
+   * @brief 回頭する
+   */
+  void run();
+
+  /**
+   * オーバーライド必須
+   * @brief 直進する際の事前条件判定をする
+   * @param targetValue 回頭の条件値　色または回頭角度
```

Comment:
```suggestion
   * @brief 直進する際の事前条件判定をする
   * @param targetValue 回頭の条件値　色または回頭角度
   * @note オーバーライド必須
```
の方がいいかな。@remarkでもいいけど。

https://cercopes-z.com/Doxygen/cmd-brief-dxy.html